### PR TITLE
feature: add routes list page and i18n content

### PR DESCRIPTION
**Pull request by bzp2010**
> add routes list page and i18n content

> 添加路由列表页面及其相关的国际化内容

### CODE REVIEW

File: src/locales/en-US/menu.ts [hunk 1]
```diff
@@ -53,4 +53,8 @@ export default {
   'menu.ssl.list': 'SSL List',
   'menu.ssl.edit': 'Edit',
   'menu.ssl.create': 'Create',
+  'menu.routes': 'Route',
+  'menu.routes.list': 'Routes List',
```
Comment (juzhiyuan) [inline src/locales/en-US/menu.ts:57]: Route List

File: src/locales/zh-CN/menu.ts [hunk 2]
```diff
@@ -53,4 +53,8 @@ export default {
   'menu.ssl.list': 'SSL 列表',
   'menu.ssl.edit': '编辑',
   'menu.ssl.create': '创建',
+  'menu.routes': 'Route',
```
Comment (juzhiyuan) [inline src/locales/zh-CN/menu.ts:56]: 路由

**Comment by bzp2010** [comment]
How can I see the todo list?

**Comment by juzhiyuan** [comment]
> Ok, but where can I see the todo list?

https://github.com/apache/incubator-apisix-dashboard/projects/1

**Comment by juzhiyuan** [comment]
When you choose your issue, please let me know :) We could make a discussion before your contribution.

**Comment by bzp2010** [comment]
Ok, I will check list to choose issus.

**Comment by juzhiyuan** [comment]
The new route will be updated next week.

**Pull request closed without merging**